src/Razor/tools/RazorExtensionsTransport/RazorExtensionsTransport.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Transport package for Mvc.Razor.Extensions pdbs. For internal use only.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <IsShipping>false</IsShipping>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Mvc.Razor.Extensions" />
```

Comment:
Suggest adding `<ReferenceOutputAssembly>false</ReferenceOutputAssembly>` or `ReferenceOutputAssembly="false"`.

Disabling compilation of this project would be great too because package will otherwise contain a useless DLL. See src\Servers\IIS\AspNetCoreModuleV2\Symbols\Microsoft.AspNetCore.ANCMSymbols.csproj for an example.